index.js

```diff
@@ -67,7 +91,18 @@ function getAverageIMDBRating() {}
  *  countByRating(movies);
  *  //> { G: 3, PG: 7 }
  */
-function countByRating() {}
+function countByRating(movies) {
+  let final = {}
+  for (let movie of movies){
+    if (final.movie > 1){
```

Comment:
We were very close here - notice that our conditional focuses on `movie` - which is a _massive_ object.  Then we compare `final.movie` to the number `1` - the issue is final.movie is an object - which means we can't set it as a key in another object.